extensions/amp-analytics/0.1/vendors.js

```diff
@@ -963,6 +963,13 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
       'metric': 'cm',
     },
     'optout': '_gaUserPrefs.ioo',
+    'linkers': {
+      '_gl': {
+        'ids': {
+          '_ga': 'CLIENT_ID(_ga)',
```

Comment:
oh, this should actually be `CLIENT_ID(AMP_ECID_GOOGLE,,_ga)`

Reply:
`${clientId}` is even better

Reply:
Done.